index.html

```diff
@@ -5,11 +5,22 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <meta name="description" content="Play incredible Star Wars Quiz! Can you beat the others? CLICK TO PLAY!">
+  <title>Star Wars Quiz</title>
+  <link rel="stylesheet" href="styles/index.css" />
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="app">
+    <div id ="main-containter" class="main-bg-image">
+      <div class="sw-logo"></div>
+      <div class="quiz-main-menu">
```

Comment:
you can change 'div' to the 'nav' tag

Reply:
It's ok I think.

Reply:
put only quiz-main-menu in the nav tag. main-grid-container should be div. https://stackoverflow.com/questions/11751017/can-i-have-a-logo-image-inside-a-nav-element

Reply:
I'm also in favor of switching the tags. `#main-grid-container` would do better as a `div` and `.quiz-main-menu` as a `nav`.

Reply:
I just made mistake here, I was sure it was correct.